### PR TITLE
add flask as backend for restful api calls from chrome extension fron…

### PR DESCRIPTION
**Pull request by guanw**
…t-end

set up flask framework

### CODE REVIEW

File: flask_backend/server.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+@app.route('/webpage', methods=['POST'])
+def send_webpage():
+    return "success"
+
+
+@app.route('/detect', methods=['POST'])
+def detect():
+    webpage = request.form['data']
+    print webpage
+
```
Comment (passByReference) [inline flask_backend/server.py]: What does detect do? Print values of any 'data' attribute?

File: chrome_extension/content.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// this file execute in the context of webpage user is currently look at. Will only execute once
+// and post html source page to flask backend for further processing
+
+xmlhttp=new XMLHttpRequest();
+xmlhttp.onreadystatechange=function(){
+  if (xmlhttp.readyState==4 && xmlhttp.status==200){
```
Comment (passByReference) [inline chrome_extension/content.js]: What does readyState 4 mean?

File: chrome_extension/manifest.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// this is the mandatory configuration file for chrome extension app
```
Comment (passByReference) [inline chrome_extension/manifest.json]: Could you add a link to the configuration page? Thanks~

**Comment by guanw** [comment]
Updated the code in the morning, check again.

**Pull request merged**